src/types/relocatable.rs

```diff
@@ -687,6 +687,43 @@ mod tests {
         );
     }
 
+    #[test]
+    fn relocatable_add_i32() {
+        let reloc = relocatable!(1, 5);
+
+        assert_eq!(&reloc + 3, relocatable!(1, 8));
+        assert_eq!(&reloc + (-3), relocatable!(1, 2));
+    }
+
+    #[test]
+    #[should_panic]
```

Comment:
Issue created
https://github.com/lambdaclass/cairo-rs/issues/547

Reply:
Can't an error check be used here instead of `#[should_panic]`?

Reply:
Not really, since the function is panicking instead of throwing an error